src/server/scripts/Spells/spell_shaman.cpp

```diff
@@ -265,8 +265,13 @@ class spell_sha_earth_shield : public SpellScriptLoader
 
             bool CheckProc(ProcEventInfo& eventInfo)
             {
-                if (!eventInfo.GetDamageInfo() || !HasEffect(EFFECT_1) || eventInfo.GetDamageInfo()->GetDamage() < GetTarget()->CountPctFromMaxHealth(GetEffect(EFFECT_1)->GetAmount()))
+                if (!eventInfo.GetDamageInfo() || !HasEffect(EFFECT_1))
                     return false;
+
+                if (AuraEffect* effect = GetEffect(EFFECT_1))
```

Comment:
False positive - HasEffect takes care of it (and more since it not only checks that it exists on aura but also that it is applied on current target). Don't fix.

Reply:
@DDuarte